<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileCalDialog</class>
 <widget class="QDialog" name="ProfileCalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="savedata">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>560</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>310</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>填入下面，格式如下</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>290</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请将剩下的井点从左到右按顺序标记，并将坐标</string>
   </property>
  </widget>
  <widget class="QToolButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>530</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>220</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Z1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>190</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请将在图中Z坐标尺选择两个点，并填写其深度</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>40</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="deepth1">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> -16000 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deepth2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>250</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> -28000 </string>
   </property>
  </widget>
  <widget class="QToolButton" name="openfile">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>打开连井剖面图</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>701</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>380</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="plainText">
    <string>15263550.05 4965333.35 井1
15270689.93 4965651.65 井2
15277180.23 4966450.02 井3
15282490.27 4966610.31 井4
15286231.42 4966455.20 井5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>250</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Z2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
